### ISSUE TITLE
get_chats(): Can't deserialize tdlib data

### ISSUE THREAD

**Issue by ibortnik** [aCLr/rust-tdlib#12]
Hello,

Thank you for the crate. I am getting this error when using `get_chats`.

```rust
let chats = client1
    .get_chats(GetChats::builder().limit(10).build())
    .await
    .unwrap();
```

```
 ERROR rust_tdlib::client::worker       > can't deserialize tdlib data: can't deserialize for updateSupergroupFullInfo with error: missing field `invite_link`
 ERROR rust_tdlib::client::worker       > can't deserialize tdlib data: can't deserialize for updateBasicGroupFullInfo with error: invalid type: map, expected a string
 ERROR rust_tdlib::client::worker       > can't deserialize tdlib data: can't deserialize for updateSupergroupFullInfo with error: invalid type: map, expected a string
 ```

**Comment by antonio-antuan** [comment]
What's the version of your tdlib library?

пн, 19 июл. 2021 г., 22:15 ibortnik ***@***.***>:

**Comment by ibortnik** [comment]
1.7.0

**Comment by antonio-antuan** [comment]
Hm, that's weird because `GetChats` returns just list of ids...
Could you try this:
```
client
        .get_chats(
            GetChats::builder()
                .limit(10)
                .offset_order(9223372036854775807)
                .build(),
        )
        .await.unwrap();
```

**Comment by antonio-antuan** [comment]
Also, you can set RUST_LOG=trace and provide me log file... But beware, there can be private data

**Comment by antonio-antuan** [comment]
And yes, you can grep that logs by `updateSupergroupFullInfo` and provide just matched rows

**Comment by ibortnik** [comment]
Thanks! Using `offset_order(i64::MAX)` worked.

What is the problem?

**Comment by antonio-antuan** [comment]
Well, I don't even know :D 
I just tried to reproduce the same error and found that if `offset_order` not set - response not returned and code become blocked.
I need to dive into two problems: deserialize error and blocked `get_chats` request.
Consider not to close issue for now and get to check it in several days.

**Comment by antonio-antuan** [comment]
Good news everyone. Year has not passed and the issue is resolved with #14.
